### DISCUSSION TITLE
Browser does not redirect me to my GitHub desktop

### DISCUSSION THREAD

**Discussion by Noorullah-Zadran** [community/community#76685] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hi every one, i am new to GitHub and i have problem signing in to my GitHub desktop, i tried reinstalling the App, cleared the browser cache and enabled cookies but still the browser does NOT redirecting me to my GitHub desktop, anyone knows? please help!, Thanks!

**Comment by ghost** [comment]
Look into this [desktop/desktop#11410](https://github.com/desktop/desktop/issues/11410) you might resolve your issue.

**Reply by ghost** [reply]
@Noorullah-Zadran Have you tried changing your default browser to Edge?

**Reply by Noorullah-Zadran** [reply]
yes, i changed my default browser even later i reinstalled google chrome but no work!

**Reply by ghost** [reply]
try [this](https://github.com/desktop/desktop/issues/11410#issuecomment-811442461)

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @Noorullah-Zadran! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Comment by Noorullah-Zadran** [comment]
i am very frustrated for the past 10 days trying to sign in to my GitHub
desktop but i keep failing, i tried too much things but no solution.
is there any chance for me to sign in to my GItHub desktop OR i must say
Good bye to GItHub please reply. thanks!

**Comment by pathakjiop** [comment]
Just reinstall github desktop and make sure the path is correct.

**Comment by Noorullah-Zadran** [comment]
I did that but it still does NOT work.

**Reply by pathakjiop** [reply]
did you cheak the path then..

**Comment by Noorullah-Zadran** [comment]
no, kindly guide me how can i check the path, thanks

**Reply by pathakjiop** [reply]
Go to the github desktop folder on your pc and then run the update.exe file
Then try again

**Comment by Noorullah-Zadran** [comment]
Thanks for your time but i checked that link as well there were only that
problem with no solution. it has been ten days i have been suffering this
pain and i reached to the conclusion that i must not go further into this
headache. Github is a big piece of shit and i wonder how people get benefit
from it if someone like me spending 10 days to log in but can NOT and
people using it...... strange for me to digest. Thanks thgouh.

**Comment by Amilashan** [comment]
response #1

**Comment by awirnr** [comment]
Response #2
